### ISSUE TITLE
username for popcorntime.io

### ISSUE THREAD

**Issue by genecyber** [tradle/craigslist-on-chain#2]
Not sure how to get access

**Comment by mvayngrib** [comment]
@genecyber not sure what you mean by access? This repurposes Popcorn Time's UI for adding blockchain-hosted identities to the Craigslist experience

**Comment by genecyber** [comment]
the build proccess does a pull of the popcorntime codebase
this requires a username and password to do this pull....

**Comment by mvayngrib** [comment]
ah interesting, that wasn't the case before. May I ask, are you interested in this as a developer or as an end user? This is a work in progress: the UI is at the stage where it can browse Craigslist similarly to how Popcorn Time browses movies/shows. This project is now waiting for the [blockchain-based part](https://github.com/tradle/tim) to crystallize, which should be soon (think weeks).

If you're interested in getting involved, there is work to be done in both departments (UI, blockchain), and I can outline some todos.

If you want to see what it looks like so far, I'll make the latest build available, let me know.

**Comment by genecyber** [comment]
All of the above. Mostly as a developer.  For the past few months I have been working on a full browser-based decentralized and peered wallet. While trying to get a handle on DHT I came across you guys and the paper on hardening DHT. Which ultimately led to this particular implementation. I was hoping to get something up so that I can take a look at your end to end stack. 

I'm pretty hooked on writing just to be applications at this point. And would love to help you guys round out stuff. 
 I don't so love to see a working copy of the craigslist on chain.

**Comment by mvayngrib** [comment]
@genecyber we'd love for you to get involved. Want to hop on a quick Skype call to see what would be most interesting for you to work on? My id is **luigimvm**.

Meanwhile, you can jump in and start submitting issues and PRs to our repos, and/or you can tell me what parts you'd be interested in working on:
1. [blockchain and p2p components](https://github.com/tradle/tim) - e.g. we need an SPV wallet instead of our current common-blockchain one. [bcoin](https://github.com/indutny/bcoin) is a candidate. Does the wallet you're working on support SPV?
2. UI-related parts
   - [react-native frontend](https://github.com/pgmemk/tim) - our react native frontend. I see there's no readme there, @pgmemk can you add at least a basic one with a couple of screenshots?
   - react native components - we currently have [react-native-udp](https://github.com/tradle/react-native-udp) and [react-native-level](https://github.com/tradle/react-native-level) (and [react-native-level-fs](https://github.com/tradle/react-native-level-fs)). One thing needed there is getting automated testing set up (in react-native's env).

**Comment by genecyber** [comment]
Looking at Trust in Motion reminds me of at I'm building. My staging server is here: http://tech.ribbit.me/vault/ 

It works best in chrome, and is a mess of experimentation so it's buggy. But... 

Upon launch it creates an identity hdkey and stores it locally. It then connects to a mesh made via webrtc

JavaScript console had lots exposed globally to allow a sandbox

meshnet is the dht object
newtables is the local storage object 
For example newtables.privkey.newHD lets you make a new private key and tag it as an identity

Chat is a fun object too. 

var chat = new Chat({payload: "my chat msg"}) 

If you make the chat you can now view the hash and it will auto sign with an identity address

chat.addToChainTip() adds the chat msg to the chat chain

chat.broadcast() sends it over the dht

And so on.

**Comment by genecyber** [comment]
Common blockchain like Wei lu from hive? I helped inspire her for that while I was creating symchain. 

I can't do a Skype till tomorrow but I'd love to. I'll reach out tomorrow. 

I'm currently using insight Apis for getting utxo's and broadcasting but plan to switch soon. I have loaded up in my wallet a fully browserfied spv wallet. It's using a webrtc / Tcp bridge. I have libs to do spv I just haven't had time to do it.

**Comment by mvayngrib** [comment]
yep! I met @weilu in SF, but found common-blockchain separately. Small world :)

looks like you've done a lot of work in this space. Can you paste a few links to your most relevant repos? And absolutely let's chat tomorrow.
